lib/comgr/CMakeLists.txt

```diff
@@ -185,16 +185,31 @@ add_subdirectory(yaml-cpp EXCLUDE_FROM_ALL)
 include_directories(./yaml-cpp/include)
 
 set(CLANG_LIBS
-  clangFrontendTool)
+  clangFrontendTool
+  clangFrontend
```

Comment:
These are all transitive dependencies (`INTERFACE_LINK_LIBRARIES`, in CMake's terminology) of `clangFrontendTool`. In what configuration do you see link errors when these are omitted?